### PR TITLE
feat(editors/subscriber-later-binding): Add plugin with filterable FCDA list

### PR DESCRIPTION
**Pull request by juancho0202**
Refactor to have a single Later Binding left panel with filterable FCDA list that may be used either for SVC or GSEControl elements

(WIP)

Missing:

- Adding Later Binding Goose tests
- Updating Later Binding Sample Value tests
- Removing unused code once the team approves the refactor

### CODE REVIEW

File: src/editors/SMVSubscriberLaterBinding.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { css, html, LitElement, property, TemplateResult } from 'lit-element';
 
-import './subscription/smv-laterbinding/svc-laterbinding-list.js';
```
Comment (dlabordus) [inline src/editors/SMVSubscriberLaterBinding.ts:3]: Shouldn't we cleanup this old file, including translations?

File: src/editors/GooseSubscriberLaterBinding.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      </div>
+    </div>`;
+  }
+
+  static styles = css`
+    :host {
+      width: 100vw;
+    }
+
+    .container {
```
Comment (dlabordus) [inline src/editors/GooseSubscriberLaterBinding.ts:27]: I think @JakobVogelsang likes the list be the full width for this issue and when picking up the next issue to add the second list make it 50%.

So for now that means removing 'display' and 'height' from the class 'container' and also the class 'column' isn't needed for now. When picking up issue #865, these lines can be added again.

Comment (JakobVogelsang) [reply src/editors/GooseSubscriberLaterBinding.ts:27]: That is very true :)

**Review by dlabordus — CHANGES_REQUESTED** [review]

**Review by dlabordus — APPROVED** [review]

**Comment by juancho0202** [comment]
Closes #860

**Pull request merged**